### PR TITLE
use shorter dir/file names like "s1reader"

### PR DESCRIPTION
**Pull request by yunjunz**
+ rename the repo directory and core module names to shorter ones, as discussed in https://github.com/opera-adt/s1-reader/discussions/14. 

+ update the corresponding usages due to the renaming

### CODE REVIEW

File: bin/s1_read.py [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import os
 import sys
 
-from sentinel1_reader import sentinel1_reader, sentinel1_orbit_reader
+from s1reader import reader, orbit
```
Comment (LiangJYu) [inline bin/s1_read.py:4]: I'm concerned renaming `sentinel1_reader, sentinel1_orbit_reader` to `reader, orbit` this too vague. @yunjunz What do you think about the following?
```suggestion
from s1reader import s1_reader, s1_orbit
```
 `s1_` is still much shorter than `sentinel1_` and it leaves no ambiguity as to purpose/origin.

Comment (yunjunz) [reply bin/s1_read.py:4]: yeah, that sounds a good idea.

Comment (yunjunz) [reply bin/s1_read.py:4]: I will adjust the naming in the PR accordingly to this.

Comment (yunjunz) [reply bin/s1_read.py]: Done.

**Review by LiangJYu — APPROVED** [review]
LGTM. I really like these changes.

**Review by vbrancat — APPROVED** [review]
LGTM. Thanks for these changes @yunjunz. Your nomenclature looks really coincise. We should make sure that none of the modules using the reader (e.g. COMPASS) will break after mergin this PR>

**Review by hfattahi — APPROVED** [review]
LGTM. Thanks Yunjun.

**Pull request merged**